### PR TITLE
fix: set default for examples to be non secure mode

### LINKED ISSUE #37: Add setting EDGEX_SECURITY_SECRET_STORE=false to example code
The example are often run native (not docker) and will return error is user doesn't first set `EDGEX_SECURITY_SECRET_STORE=false`. Problem is the user doesn't know to do this and then we get questions in the Slack channels.

So for just the examples that never need Secret Store (this excludes the secrets example), we should add setting `EDGEX_SECURITY_SECRET_STORE=false` as the first thing in the main() with a Warning comment that it should be removed for non-example code.

### PR DESCRIPTION
**Pull request by rsdmike**
closes #37 
Signed-off-by: Mike <michael.johanson@intel.com>

### CODE REVIEW

File: app-services/advanced-filter-convert-publish/main.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ const (
 )
 
 func main() {
+	os.Setenv("EDGEX_SECURITY_SECRET_STORE", "false")
```
Comment (lenny-goodell) [inline app-services/advanced-filter-convert-publish/main.go:35]: We should add a comment, on all these, that this is non-standard just for the example to run w/o user having to set this environment variable. Should not be use in real world code.....

File: app-services/simple-filter-xml/Dockerfile [hunk 2]
```diff
@@ -1,46 +1,48 @@
...
-
-LABEL license='SPDX-License-Identifier: Apache-2.0' \
-  copyright='Copyright (c) 2019: Intel'
-
-RUN apk --no-cache add zeromq
-
-COPY --from=builder /app-service-examples/app-services/simple-filter-xml/res /res
-COPY --from=builder /app-service-examples/app-services/simple-filter-xml/app-service /simple-filter-xml
-
+#
```
Comment (lenny-goodell) [inline app-services/simple-filter-xml/Dockerfile:1]: Why is this changed and can't determine what was changed...

**Review by lenny-goodell — CHANGES_REQUESTED** [review]

File: app-services/simple-filter-xml/Dockerfile (hunk 2 above)
Comment (rsdmike) [reply app-services/simple-filter-xml/Dockerfile:1]: line 43

Comment (lenny-goodell) [reply app-services/simple-filter-xml/Dockerfile:1]: K, see it now. This is not needed since it is already done in the code.

**Review by lenny-goodell — CHANGES_REQUESTED** [review]

File: app-services/simple-filter-xml/Dockerfile (hunk 2 above)
Comment (rsdmike) [reply app-services/simple-filter-xml/Dockerfile:1]: i did to show how it can be done in docker too - i dont think it hurts.

Comment (lenny-goodell) [reply app-services/simple-filter-xml/Dockerfile:1]: K, got it. ;-)

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Pull request merged**